client/actions/form.js

```diff
@@ -1,68 +1,83 @@
 "use server";
 
-import { redirect } from "next/navigation";
 import { revalidatePath } from "next/cache";
 import { serverUrl } from "@/constants";
 
 // CREATE NEW APPT
 export async function requestAppt(formValues) {
-  const response = await fetch(serverUrl + "appointments", {
-    method: "POST",
-    headers: { "Content-Type": "application/json" },
-    body: JSON.stringify(formValues),
-  });
-  const data = await response.json();
+  try {
+    const response = await fetch(serverUrl + "appointments", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(formValues),
+    });
+    const data = await response.json();
 
-  if (!response.ok) {
-    return { message: data.message };
+    if (!response.ok) {
+      return { message: data.message };
+    }
+  } catch (error) {
+    throw error;
   }
```

Comment:
This try catch does nothing, unless I'm reading it wrong

Reply:
Good catch, this is redundant